earth_enterprise/rpms/build.gradle

```diff
@@ -89,34 +89,6 @@ def rpmPlatformString = org.opengee.os.Platform.rpmPlatformString
 // Create a file handle for gee_long_version_txt
 def openGeeVersionFile = new File( gee_long_version_txt )
 
-// Call scons to create the file 'gee_long_version.txt'
-// The actual parameter used by scons is based on the parameter
-// passing into Gradlew
-if ( !openGeeVersionFile.exists() ) {
-    GeeCommandLine.expand(
-      ["/usr/bin/scons", scons_option, gee_long_version_txt ],
-      "Runing `scons` to produce <${gee_long_version_txt}> failed!",
-      null, // Use default environment variables.
-      new File( src_dir )
-    )
-}
```

Comment:
Most of the time this is just a test if a file exists.  I don't think that's too expensive.

The `scons` code to generate `version.txt` is also not that expensive.  It reads Git tags and loops through them, but I doubt it would take a second.

Reply:
True if you already have done a build and stage this is really fast...  But should I have to do a build and stage just to run `gradle tasks`? or `gradle clean`?  *That* seems like an excessive thing to have to do for those two operations.

Reply:
As I mentioned, the old code ran neither a build, nor a stage to get a version.  It just ran the `scons` code to generate the version file.  I haven't timed it, but I don't think that would take even a second to run.